modules/ROOT/pages/dataweave-variables.adoc

```diff
@@ -151,6 +151,67 @@ Note that DataWeave variables cannot be reassigned. They are also distinct from
 
 You can combine several local variable definitions as a comma separated list inside the using function. For example: `using (firstName='Annie', lastName='Point')`
 
+== Naming Rules for Variables
+
+Variable names have certain requirements to be valid, following are the rules for naming variables:
+
+* All variable names must begin with a letter of the alphabet (a-Z).
+* After the first letter, variable names can contain any combination of letters, numbers and underscores (`_`).
+* The variable name cannot be any DataWeave reserved keyword (see <<reserved-keywords>> for a complete list).
+
+Below are some examples of valid variable names:
+
+* `myType`
+* `abc123`
+* `a1_3BC_22`
+* `Z_______4`
+* `F`
+
+The following table has some examples of invalid variable names, and a description of what makes them invalid:
```

Comment:
suggest "has some examples" -> "provides examples" 

"names, and" -> "names and"